data/dictionary/values getter by keys/83ac4d5d-9ac9-48d0-ae17-a5a77b18cfb6.json

```diff
@@ -1,5 +1,5 @@
 {
-  "description": "Retrieves multiple items' values from the dictionary by their keys, in the order of the keys provided.\nIf any of the items are not found in the dictionary, a 'null' value will be sent in its place.\n\nExample A:\n1. { \"first\": 1, \"third\": 3, \"fifth\": 5 } @0 received via `dict`\n2. [ \"first\", \"third\" ]@0 received via `key`\n3. `values` sends  [ 1, 3 ]@0\n\nExample B:\n1. { \"first\": 1, \"third\": 3, \"fifth\": 5 } @0 received via `dict`\n2. \"second\"@0 received via `key`\n3. `values` sends  [ 1, null ]@0\n\nMore:\nhttps://github.com/Cranq-io/cranq-tutorials/tree/main/reference/3_querying_data/3_1_getters",
+  "description": "Retrieves multiple items' values from the dictionary by their keys, in the order of the keys provided.\nIf any of the items are not found in the dictionary, a 'null' value will be sent in its place. \n\nExample A:\n1. { \"first\": 1, \"third\": 3, \"fifth\": 5 } @0 received via `dict`\n2. [ \"first\", \"third\" ]@0 received via `key`\n3. `values` sends  [ 1, 3 ]@0\n\nExample B:\n1. { \"first\": 1, \"third\": 3, \"fifth\": 5 } @0 received via `dict`\n2. [\"first\",\"second\"]@0 received via `key`\n3. `values` sends  [ 1, null ]@0\n4. `not found jeys` sends ['second']@0\n\nMore:\nhttps://github.com/Cranq-io/cranq-tutorials/tree/main/reference/3_querying_data/3_1_getters",
```

Comment:
typo in Example B/4. "not found **j**eys"

Reply:
in both example the port `keys` is referred mistakenly as `key`

Reply:
The reference link is not working on Github

Reply:
Done